src/main/kotlin/com/dansplugins/factionsystem/lang/Language.kt

```diff
@@ -8,7 +8,7 @@ import java.util.*
 
 class Language(plugin: MedievalFactions, private val language: String) {
 
-    private val resourceBundles: List<ResourceBundle>
+    private var resourceBundles: List<ResourceBundle>
```

Comment:
This doesn't need to be a var - in fact, since we're using the try as an expression, you can join the declaration and assignment (variables and init blocks are interleaved on class instantiation and will be run in the order they're written)

Reply:
Changed this back to a val!